bundles/org.openhab.binding.argoclima/README.md

```diff
@@ -65,13 +65,13 @@ The same values apply to **both** `remote`  and `local`.
 | Name                   | Type            | Description                                                                                                                                                                                                                                                     | Default                              | Required | Advanced |
 |------------------------|-----------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|----------|----------|
 | schedule1DayOfWeek     | text(multiple)  | Days (set comprising of values  ```MON```, ```TUE```, ```WED```, ```THU```, ```FRI```, ```SAT```, ``SUN``), when Schedule Timer 1 actions should be performed. This is used only if ```active-timer``` [channel](#channels) is in ```SCHEDULE_TIMER_1``` mode.  | [MON, TUE, WED, THU, FRI, SAT, SUN]  | no       | yes      |
-| schedule1OnTime        | text            | The time of day (HH:MM) the device should turn **ON** *(in the last used mode)* on the ```schedule1DayOfWeek```-specified days. In effect only if ```active-timer``` [channel](#channels) is in ```SCHEDULE_TIMER_1``` mode                                     |  8:00                                | no       | yes      |
+| schedule1OnTime        | text            | The time of day (HH:MM) the device should turn **ON** _(in the last used mode)_ on the ```schedule1DayOfWeek```-specified days. In effect only if ```active-timer``` [channel](#channels) is in ```SCHEDULE_TIMER_1``` mode                                     |  8:00                                | no       | yes      |
```

Comment:
Might as well use 1 back tick \` instead of 3 ```? Doesn't the action complain about this?

Reply:
```suggestion
| schedule1OnTime        | text            | The time of day (HH:MM) the device should turn **ON** _(in the last used mode)_ on the `schedule1DayOfWeek`-specified days. In effect only if `active-timer` [channel](#channels) is in `SCHEDULE_TIMER_1` mode                                     |  8:00                                | no       | yes      |
```

Reply:
Might have missed it between the hundreds of warnings, but AFAICT it did not complain. Adapted anyway.